ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -39,7 +39,16 @@ if get_option('enable-tensorflow-lite')
     nnstreamer_filter_tflite_sources += join_paths(meson.current_source_dir(), s)
   endforeach
 
-  nnstreamer_filter_tflite_deps = [tflite_dep, thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
+  if get_option('enable-nnfw')
+    nnapi_lib = dependency('nnfw', required: true)
+    if nnapi_lib.found()
+      nnstreamer_filter_tflite_deps = [tflite_dep, thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep, nnapi_lib]
```

Comment:
how about create `nnstreamer_filter_tflite_deps` outside the `if` condition and just add `nnapi_lib` to it?